### DISCUSSION TITLE
Material for learning implementing in-situ stress and absorbing boundary

### DISCUSSION THREAD

**Discussion by miraboreasumr** [idaholab/moose#27930] [Q&A Modules: Solid mechanics]
Are there materials (couldn't find in the documents) about how MOOSE implements the in-situ stress and absorbing boundary (Lysmer et al. I guess?) Like how the in-situ stress affects the initial condition and the solving process, etc
Thanks!

**Comment by GiudGiud** [comment]
Hello

Which source file in particular? So I can find who implemented it

**Reply by miraboreasumr** [reply]
> Hello
> 
> Which source file in particular? So I can find who implemented it

I am not into the source file yet, I assume there are these features in MOOSE.

**Reply by GiudGiud** [reply]
@dschwen

**Reply by miraboreasumr** [reply]
> Hello
> 
> Which source file in particular? So I can find who implemented it

Anyone would give me some lectures about the in-situ stress in MOOSE?

**Reply by GiudGiud** [reply]
Is this in a porous-flow geomechanics context or a solid mechanics context?

**Reply by miraboreasumr** [reply]
> Is this in a porous-flow geomechanics context or a solid mechanics context?

ah, I would say more of porous-flow geomechancis.

**Reply by GiudGiud** [reply]
@1runer @cpgr do you know which doc page would fit best?
